apps/molgenis-components/src/client/graphql.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
```

Comment:
no need to wrap client , this hides the api , and if you do want to wrap it , add it to the existing client instead of adding new file

Reply:
I prefer to wrap client when using graphql. Indeed, this can be merged into client.js but for clarity I kept it seperate. No strong opinion so can merge

Reply:
I expect in long run that we gonna move the whole logic of signin/signout to the client so this is just a temp situation. Meanwhile there are good use cases to want to have convenience function for graphql (instead of bothering every coder with the details.